openfisca_france/param/param.xml

```diff
@@ -2781,7 +2784,16 @@
         <VALUE deb="2010-01-01" fin="2011-12-31" valeur="878" />
         <VALUE deb="2012-01-01" fin="2012-12-31" valeur="960" />
         <VALUE deb="2013-01-01" fin="2013-12-31" valeur="1016" />
-        <VALUE deb="2014-01-01" fin="2014-12-31" valeur="1021" />
```

Comment:
Sur le simulateur des impôts, j’ai saisi 18000 € dans AJ et 3000 €  de frais
réels dans AK : Cela donne un revenu imposable de 15000 € et une décote de
392 €. Mais on ne parle pas de la même chose tous les deux, toi tu parles du
net de la fiche de paie qui n’est pas le revenu imposable mais le net fiscal
à déclarer et moi je parle du revenu imposable calculé par le fisc en
prenant en compte notamment soit la déduction forfaitaire des 10% ou des
frais réels.

Patrick BODET

De :  adrienpacifico <notifications@github.com>
Répondre à :  openfisca/openfisca-france
<reply+001b32b27265ec8ddb570683285aae3c7b200b03805d2bec92cf00000001121a2a8c9
2a170ce026520aa@reply.github.com>
Date :  mercredi 23 septembre 2015 10:53
À :  openfisca/openfisca-france <openfisca-france@noreply.github.com>
Cc :  Patrick BODET <patrick.bodet4@wanadoo.fr>
Objet :  Re: [openfisca-france] Decote 2015 (#324)


In openfisca_france/param/param.xml
<https://github.com/openfisca/openfisca-france/pull/324#discussion_r40181930
> :
> @@ -2781,7 +2784,16 @@
>          <VALUE deb="2010-01-01" fin="2011-12-31" valeur="878" />
>          <VALUE deb="2012-01-01" fin="2012-12-31" valeur="960" />
>          <VALUE deb="2013-01-01" fin="2013-12-31" valeur="1016" />
> -        <VALUE deb="2014-01-01" fin="2014-12-31" valeur="1021" />
@iDevelopper <https://github.com/iDevelopper>  thanx. If you just fill 1AJ
(which is net imposable on your fiche de paye) you have to substract 10% as
an rebate on your business expenses. So most people will have to put 16666
(15000*0.9) euros to get your exemple. Feel free to implement it in a .yaml
test and commit it :-) !

—
Reply to this email directly or view it on GitHub
<https://github.com/openfisca/openfisca-france/pull/324/files#r40181930> .